ResearchKit/Common/ORKLocationSelectionView.m

```diff
@@ -266,8 +269,8 @@ - (void)setAnswer:(id)answer {
         }
         [_mapView setRegion:region animated:YES];
         
-        if (placemarkAnswer.addressDictionary) {
-            _textField.text = ABCreateStringWithAddressDictionary(placemarkAnswer.addressDictionary, NO);
+        if (location.address) {
```

Comment:
Should move the answer casting here since it doesn't get used until this if statement.